tests/api_tests/test_create_index.py

```diff
@@ -0,0 +1,39 @@
+from marqo.client import Client
+from marqo.errors import MarqoApiError, MarqoWebError
+import unittest
+import pprint
+from tests.marqo_test import MarqoTestCase
+
+
+class TestCreateIndex(MarqoTestCase):
+
+    def setUp(self) -> None:
+        client_0 = Client(**self.client_settings)
+        self.index_name_1 = "my-test-index-1"
+        for ix_name in ['some-bulk', self.index_name_1]:
+            try:
+                client_0.delete_index(ix_name)
+            except MarqoApiError as s:
+                pass
+
+    def test_illegal_index_name_prevented(self):
+        client = Client(**self.client_settings)
+
+        client.create_index('some-bulk')
+        try:
+            client.create_index('bulk')
+            raise AssertionError('created index with illegal name `bulk`!')
+        except MarqoWebError:
+            pass
```

Comment:
Can we be more specific about what error code was returned? `assert e.code == "something"`